environment.yml

```diff
@@ -1,15 +1,20 @@
 name: em
-
 channels:
   - conda-forge
   - defaults
 dependencies:
-  - matplotlib=3.5.2
+  - matplotlib
+  - numpy=1.26
   - git
+  - pip
   - simpeg
-  - pyvista
   - dill
-  - jupyter
   - jupyterlab
-  - ipympl
   - ipywidgets
+  - mkl-devel
+  - pkg-config
+  - cmake
+  - cython 
+  - ninja
```

Comment:
I'm going to recommend against installing all of these build related packages. It looks like they're getting installed just so you can locally build `pydiso`. Unless that's absolutely necessary I recommend installing it through `conda-forge`. Do note however that requiring `pydiso` will limit the installation to only Intel-based computers.

Alternatively you could include `pymatsolver >=0.3.0` and `python-mumps` which is available on all systems. Which would allow you to then use `pymatsolver.Mumps` in `simpeg`.